### PR TITLE
Problem Suggestion: Add "Restore Graph" to Silver - Additional Practice for USACO Silver

### PR DESCRIPTION
**Pull request by maggieliu05**
User `XRtEVBniu0RHuspDMz0rR5ux2l02` suggested adding the problem [Restore Graph](https://codeforces.com/contest/404/problem/C) to the `problems` table of the module [Silver - Additional Practice for USACO Silver](https://usaco.guide/silver/silver-conclusion#problemlist-problems).

**Automatically Generated JSON:**
```json
{
  "uniqueId": "cf-404C",
  "name": "Restore Graph",
  "url": "https://codeforces.com/contest/404/problem/C",
  "source": "CF",
  "difficulty": "Normal",
  "isStarred": false,
  "tags": [
    "Graphs",
    "DFS",
    "BFS"
  ],
  "solutionMetadata": {
    "kind": "autogen-label-from-site",
    "site": "CF"
  }
}
```
**Additional Notes**: None

*This PR was automatically generated from a user-submitted problem suggestion on the USACO guide.*

### CODE REVIEW

**Comment by envyaims** [comment]
@SansPapyrus683 for like problem suggestions could you give me codeowner because chances are i've done the problem

File: content/3_Silver/Conclusion.problems.json [hunk 1]
```diff
@@ -202,6 +202,19 @@
...
       }
     },
+    {
+      "uniqueId": "cf-404C",
+      "name": "Restore Graph",
+      "url": "https://codeforces.com/contest/404/problem/C",
+      "source": "CF",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Graphs", "DFS", "BFS"],
```
Comment (envyaims) [inline content/3_Silver/Conclusion.problems.json:212]:
```suggestion
      "tags": ["BFS"],
```

**Review by envyaims — APPROVED** [review]

File: content/3_Silver/Conclusion.problems.json (hunk 1 above)
Comment (SansPapyrus683) [reply content/3_Silver/Conclusion.problems.json:212]: does dfs not work for this problem or smth

**Comment by SansPapyrus683** [comment]
> @SansPapyrus683 for like problem suggestions could you give me codeowner because chances are i've done the problem

i only know how to give by directory

File: content/3_Silver/Conclusion.problems.json (hunk 1 above)
Comment (envyaims) [reply content/3_Silver/Conclusion.problems.json:212]: idk

**Comment by ryanchou-dev** [comment]
@SansPapyrus683 what do you think about this problem?

**Comment by SansPapyrus683** [comment]
(idk, i don't do problems anymore)

**Pull request merged**